### PR TITLE
EUPL is also an OS license.

### PR DESCRIPTION
**Pull request by waterkip**
I work on a project which has an EUPL license. I've added support in Module::Install for this. 

Cheers,
Wesley

### CODE REVIEW

File: lib/Module/Install/Metadata.pm [hunk 1]
```diff
@@ -533,6 +534,7 @@ sub __extract_license {
 		'OpenSSL License'                    => 'unrestricted', 1,
 		'SSLeay License'                     => 'unrestricted', 1,
 		'zlib License'                       => 'open_source',  1,
+		'EUPL'                               => 'eupl',         1,
```
Comment (Leont) [inline lib/Module/Install/Metadata.pm]: That would have to be 'open_source', since we have a restricted list of allowed values.

Comment (waterkip) [inline lib/Module/Install/Metadata.pm]: Could you tell me a little more about this restricted list? Thanks :)

Comment (haarg) [inline lib/Module/Install/Metadata.pm]: Module::Install uses the CPAN::Meta spec v1.4.  The list of licenses supported by this spec can be found [here](https://metacpan.org/pod/distribution/CPAN-Meta/lib/CPAN/Meta/History/Meta_1_4.pod#license).

**Comment by Leont** [comment]
Not a criticism against this patch, but most modules doing anything with license nowadays use Software::License. If you have an interest in making it accessible to other people (not sure if you are or if this was something you needed for a personal project) that may be worthwhile.

**Pull request closed without merging**